src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.ts

```diff
@@ -158,8 +160,7 @@ export class DisputantFormComponent implements OnInit, AfterViewInit {
   }
 
   onProvinceChange(province: ProvinceCodeValue) {
-    setTimeout(() => {
-      this.form.controls.address_province.setValue(province.provAbbreviationCd); // for sending two char province or state code to ARC
```

Comment:
@saikrishnametpalli-nttdata I think we should not remove this if a known province/state is selected since we're specifically setting this value to sending province abbreviation to ARC. This scenario would still be okay in terms of database rules since the function in the procedure covers this.

Reply:
As discussed with Sai, we don't need to set this value on citizen-portal since it won't be saved in the database so this can be removed safely.